README.md

```diff
@@ -58,38 +58,77 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
+There are two tutorials inside the application:
+
+- Offering service APIs via CoatRack
+- Using service APIs offered via CoatRack
+
+
+
+## Install CoatRack Locally via CLI
+
+The following prerequisites are required:
 
 * a Linux shell - Windows users can use WSL or the Git Bash shell instead
-* docker
-* docker-compose
 * OpenJDK 11
 * Maven 3.6.3 or higher
 
 
 
-There are two scripts in the `docker` directory helping you:
+Open a terminal, go into the `coatrack` directory and build all components using one the the commands below:
 
-* `build-and-push-images.sh` builds the docker image of each CoatRack modul from source and pushes the images to Dockerhub. This script shall only be used by the CI pipeline.
-* `build-and-deploy-images-locally.sh` builds CoatRack from source, creates container images on your local machine and runs CoatRack locally using these images. This is especially useful for developers who did changes to the source code and want to locally test the impact of these changes in a realistic setup.
+```sh
+bash build.sh # For Linux shells
+mvn clean package # For Linux or Windows shells
+```
 
+ 
 
+Run the Web Application component (admin):
 
-## Usage
+```sh
+java -jar spring-boot/admin/target/coatrack-admin-*.jar
+```
 
-To start using CoatRack: 
+The windows terminal does not recognize the `*` . If you are using the windows terminal then the `*` is to be replaced by the proper coatrack version number. Just take a look at the jar file in `spring-boot/admin/target/` directory and adapt the command above.
 
-- either run the admin application locally, as explained in the [Install](#install) section
-- or open the public instance available at (https://coatrack.eu).
 
-To log-in to CoatRack, a Github account is required. After logging in and filling in the registration form, the CoatRack admin application will open. Use the "Getting started tutorials" that are accessible from inside the application to learn the basics of using CoatRack.
 
-There are two tutorials inside the application:
+Now, open another session/terminal and additionally run the config-server component:
 
-- Offering service APIs via CoatRack
-- Using service APIs offered via CoatRack
+```sh
+java -jar spring-boot/config-server/target/coatrack-config-server-*.jar
+```
+
+Now you a have a fully working instance of CoatRack using non-persistent databases. You can access it on `http://localhost:8080`.
+
+
+
+## Install CoatRack Locally via Docker Compose
+
+The following prerequisites are required:
+
+* a Linux shell - Windows users can use WSL or the Git Bash shell instead
+* OpenJDK 11
+* Maven 3.6.3 or higher
+* docker 20.10.12 or higher
+* docker-compose 1.29.2 or higher
+
+
+
+With your terminal go into the `coatrack/docker` directory where you will find two scripts:
+
+* `build-and-push-images.sh` builds the docker image of each CoatRack module from source and pushes the images to Dockerhub. This script shall only be used by the CI pipeline.
```

Comment:
This does not really fit in the section considering the heading "... Locally".

Reply:
I removed the script from this section, as it is just of interest for us developers.